README.md

```diff
@@ -194,14 +193,13 @@ starting with the `v0.70.0` release (`v<Major>.<Minor>.<Release>[-<Build>]`).
 Prior to the v0.70.0 release, the scheme used was mostly `v<Major>.<Minor>`,
 where we only had minor releases (`Major = 0`).
 
-The practice of using `-agent` or `-server` is also ending with the `v0.70.0`
-release.
+The practice of using `-agent` or `-server` ended with the `v0.70.0` release.
```

Comment:
0.70 is ancient history: should we just nuke this whole confusingly historical section while we're sweeping?

Reply:
I removed the changed line here; I decided to leave the verbiage about versions (GitHub still has nearly 60 back-versions).